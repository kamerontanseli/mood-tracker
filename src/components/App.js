import React from 'react'
import { BrowserRouter, Route, Switch } from 'react-router-dom'
import NotFound from './NotFound'
import Insights from './Insights'
import Survey from './Survey'

const App = () => (
  <BrowserRouter>
    <Switch>
      <Route exact path="/" component={Insights} />
      <Route exact path="/survey" component={Survey} />
      <Route exact path="*" component={NotFound} />
    </Switch>
  </BrowserRouter>
)

export default App