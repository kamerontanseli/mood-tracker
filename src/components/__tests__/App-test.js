import React from "react";
import renderer from "react-test-renderer";
import App from "../App";

jest.mock("react-router-dom", () => ({
  Switch: p => <div className="switch" {...p} />,
  Route: p => <div className="route" {...p} />
}));

describe("render", () => {
  it("should render", () => {
    const snapshot = renderer.create(<App />);
    expect(snapshot).toMatchSnapshot();
  });
});
