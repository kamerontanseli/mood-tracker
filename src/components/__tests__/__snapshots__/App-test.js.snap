// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<div
  className="switch"
>
  <div
    className="route"
    component={[Function]}
    exact={true}
    path="/"
  />
  <div
    className="route"
    component={[Function]}
    exact={true}
    path="/survey"
  />
  <div
    className="route"
    component={[Function]}
    exact={true}
    path="*"
  />
</div>
`;
